states/pools.sls

```diff
@@ -61,9 +61,9 @@ libvirt_virsh_pool_startstop_{{ name }}:
     - run
   {% if p.ensure|default('running') == 'running' %}
     - name: virsh pool-start {{ name }}
-    - unless: virsh -q pool-list --all | grep -Eq '^{{ name }}\s+active'
+    - unless: virsh -q pool-list --all | grep -Eq '^\s?{{ name }}\s+active'
```

Comment:
please replace ``\s+`` by ``\s*``, to make backwards-compatiblity working (in case there's no space between ^ and ``{{ name }}``)